graaljs/graaljs-gradle-webpack-guide/src/main/js/main.mjs

```diff
@@ -0,0 +1,4 @@
+// GraalJS doesn't support TextEncoder yet. It's easy to add and here's a polyfill in the meantime.
```

Comment:
Since [24.2.0](https://github.com/oracle/graaljs/blob/master/CHANGELOG.md#version-2420), GraalJS does support TextEncoders. You just need to enable it with the `js.text-encoding` option.

Let's clean this up in a follow up PR.